educhain/utils/audio_utils.py

```
@@ -0,0 +1,723 @@
+        voice: Optional[str] = None,
+        api_key: Optional[str] = None,
+        **kwargs
+    ) -> Dict[str, Any]:
+        """ElevenLabs TTS implementation."""
+        try:
+            from elevenlabs import generate, save, set_api_key
+            
+            # Get API key from environment if not provided
+            api_key = api_key or os.getenv('ELEVENLABS_API_KEY')
+            if not api_key:
+                raise ValueError("ElevenLabs API key is required. Set ELEVENLABS_API_KEY environment variable or pass api_key parameter.")
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)